src/cli/javascript/js_request.rs

```diff
@@ -1,10 +1,12 @@
 use std::collections::BTreeMap;
 use std::fmt::Display;
 
+use rquickjs::{FromJs, IntoJs};
 use serde::{Deserialize, Serialize};
 
 use super::create_header_map;
 use crate::is_default;
+
 #[derive(Serialize, Deserialize, Debug)]
 #[serde(rename_all = "camelCase")]
 pub struct JsRequest {
```

Comment:
I think a better structure for JsRequest would be:

```rust
pub struct JsRequest(reqwest::Request);
```

We can then implement into_js and from_js directly this way, which would improve performance quite considerably.

Reply:
I can (probably) do that, but to be clear I will *not* be able to use the macro for it. I will implement the `JsClass` myself. The macro assumes everything inside the class is a `JsClass` as well, and in particular that requires implementing Clone, which `reqwest::Request` doesn't.

https://github.com/DelSkayn/rquickjs/blob/c6e314e84587712af9808bb1bc45222f404caf8a/macro/src/class.rs#L368

Reply:
Let's ignore the macros for now. It doesn't seem to be straight forward.